import-export-cli/integration/testutils/policy_testUtils.go

```diff
@@ -75,6 +76,49 @@ func ValidateThrottlePolicyExportImport(t *testing.T, args *PolicyImportExportTe
 	validatePoliciesEquality(t, true, args, importedPolicy)
 }
 
+// Validates whether the throttling policy deletion is complete
+func ValidateThrottlingPolicyDelete(t *testing.T, args *PolicyImportExportTestArgs, username, password, policyType string) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	base.WaitForIndexing()
+
+	policyName := fmt.Sprintf("%v", args.Policy[policyNameKey])
+
+	_, err := deleteThrottlingPolicy(t, policyName, policyType, args)
+
+	assert.Nil(t, err, "Error while deleting the API Policy")
+
+}
+
+// Validates whether the throttling policy deletion not exists
+func ValidateThrottlingPolicyDeleteNotExists(t *testing.T, args *PolicyImportExportTestArgs, username, password, policyType string) {
```

Comment:
Here as well. Why do we need username and password as arguments here? I can't find the usages of those inside this function

Reply:
Fixed by [859ba91](https://github.com/wso2/product-apim-tooling/pull/928/commits/859ba919a58e2b9a690c34ebf23a137abba6f93b)